src/BDCS/API/V0.hs

```diff
@@ -1303,7 +1306,13 @@ instance FromJSON PackageNEVRA where
 
 -- Make a PackageNEVRA from a tuple of NEVRA info.
 mkPackageNEVRA :: (T.Text, Maybe T.Text, T.Text, T.Text, T.Text) -> PackageNEVRA
-mkPackageNEVRA (name, epoch, version, release, arch) = PackageNEVRA name epoch version release arch
+mkPackageNEVRA (name, epoch, version, release, arch) = PackageNEVRA name (epoch' epoch) version release arch
+  where
+    epoch' Nothing = Nothing
```

Comment:
This can be simplified to:

    epoch' (Just e) = Just ((read $ T.unpack e) :: Int)